circle.yml

```diff
@@ -0,0 +1,23 @@
+machine:
+  timezone: UTC
+  java:
+    version: oraclejdk8
+  environment:
+    _JAVA_OPTIONS: "-Xms512m -Xmx1024m"
+dependencies:
+  override:
+    - mvn dependency:go-offline --fail-never
```

Comment:
Got it. Will remove the section and rebase the branch against master after #1882 merged.